### DISCUSSION TITLE
Unable to drag drop using `data-*`

### DISCUSSION THREAD

**Discussion by shashankvivek** [cypress-io/cypress#16051] [Questions and Help]
I have two components. I referred to [cypress example](https://github.com/cypress-io/cypress-example-recipes/blob/0364b2fff1f9a49c71b036588b14619b61b56171/examples/testing-dom__drag-drop/cypress/integration/drag_n_drop_spec.js#L88) to do the drag drop. 

```
    cy.get('[data-app-id=S000001]').trigger('dragstart');
    cy.get('[data-date=1619433439916]').trigger('drop');
```

but this does not seem to work. 

I do not rely css to try [jQuery-UI-Sortable](https://docs.cypress.io/api/commands/trigger#jQuery-UI-Sortable)

what am I doing wrong ?

**Comment by bahmutov** [comment]
It is impossible to say what you are doing wrong, or how the application is supposed to work without a small reproducible example. Please provide the example we can run for us to debug and help you.

**Comment by shashankvivek** [comment]
@bahmutov : Here is the stackblitz example of the UI which I am creating. I need to perform the drag and drop for the calendar. 

Demo : https://stackblitz.com/edit/angular-external-drag-and-drop?file=app.component.ts
